src/Sylius/Behat/Context/Cli/ShowingAvailablePluginsContext.php

```diff
@@ -45,10 +46,10 @@ public function __construct(KernelInterface $kernel)
     public function runShowAvailablePluginsCommand(): void
     {
         $this->application = new Application($this->kernel);
-        $this->application->add(new SetupCommand());
+        $this->application->add($this->setupCommand);
 
-        $this->command = $this->application->find('sylius:show-available-plugins');
-        $this->tester = new CommandTester($this->command);
+        $this->setupCommand = $this->application->find('sylius:show-available-plugins');
```

Comment:
As far as I see, `$application` is only used to find the setup command. We could move a few of these lines to the constructor and simplify the implementation a little bit:

```php
	/** @var CommandTester */
	private $tester;

	/** @var SetupCommand */
	private $setupCommand;

    public function __construct(KernelInterface $kernel, SetupCommand $setupCommand)
    {	
		$application = new Application($kernel);
        $application->add($this->setupCommand);

        $this->setupCommand = $this->application->find('sylius:show-available-plugins');
    }

    /**
     * @When I run show available plugins command
     */
    public function runShowAvailablePluginsCommand(): void
    {
        $this->tester = new CommandTester($this->setupCommand);
        $this->tester->execute(['command' => 'sylius:show-available-plugins']);
    }
```